### DISCUSSION TITLE
Several changes to contacts that are basically untouched

### DISCUSSION THREAD

**Discussion by My1** [tine20/tine20#7378] [Q&A]
so the users access the tine using 3 ways, the webinterface, thunderbird which also has [an Extension for CardDav syncing](https://addons.thunderbird.net/de/thunderbird/addon/cardbook/) and their phones (mostly Samsung Android phones. and apparently there are a ton of changes posted to the contacts. many things are qithout consequence and look more like an edit war where one clears several fields on a contact and another comes in later and puts those fuelds in.

what also sometimes happens according to the editlog is that fields allegedly change, but the start and end value is the same,

one thing that isnt so unconsequential tho is that birthdays seem to change and sometimes disappear completely which is more annoying, especially if a contact hasnt been touched by a user themselves. Tine Version is Amon (2022.04.02.120)

**Comment by pschuele** [comment]
hi @My1 ,

I think there might be a problem/bug lurking there ... we should try to find out if this can be fixed.

can you give an example? a DEBUG log would be great!
